CUDAEnv/Flow.jl

```diff
@@ -39,23 +53,14 @@ struct Flow{D, V, S, F, B, T}
     U :: NTuple{D, T}  # domain boundary values
     Δt:: Vector{T}  # time step
     ν :: T                  # kinematic viscosity
-    function Flow(N::NTuple{D}, U; Δt=0.25, ν=0., uλ::Function=(i, x) -> 0., T=Float64) where D
+    function Flow(N::NTuple{D}, U; Δt=0.25, ν=0., uλ::Function=(i, x) -> 0., f=identity, T=Float64) where D
```

Comment:
Don't you need to change all the `adapt!` calls below to `f` calls?

Reply:
Please see the PR comment on why I am not using it yet.

Reply:
Sorry, I misunderstood the comment.